drivers/net/ethernet/cadence/macb.c

```diff
@@ -220,6 +220,7 @@ static void macb_get_hwaddr(struct macb *bp)
 
 static int macb_mdio_read(struct mii_bus *bus, int mii_id, int regnum)
 {
+#if false
```

Comment:
You are right. It is ugly. My motivation to do so was that upstream changes would still be visible...